.github/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -0,0 +1,76 @@
+name: Bug Report
+description: File a bug report
+labels: [kind/bug, state/triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **_Are you a customer of Octopus Deploy? Don't raise the issue here. Please contact [our support team](https://octopus.com/support) so we can triage your issue, so that we can make sure it's handled appropriately._**
+  - type: checkboxes
+    id: team
+    attributes:
+      label: Team
+      description: Assign a `team/<team>` label to this issue so it can be dispatched to them. Use the [ownership map](https://whimsical.com/ownership-map-NzbiD4HJyvhC9jNJNfS6TG) to find the most appropriate team to assign this issue to. If you are not sure, or they are not listed, choose `team/fire-and-motion`
+      options:
+        - label: I've assigned a team label to this issue
+          required: true
+  - type: input
+    id: severity
+    attributes:
+      label: Severity
+      description: Is this blocking customers? How many instances are affected? Is there some context that will help us prioritize?
+      placeholder: Blocking a handful of customers, though the workaround is not great, it'll be fine for a couple of weeks
```

Is the free text too confusing? Would it be better to have a series of checkboxes?

I worry we'd lose some nuance if we did (e.g. expected timeframe).

```suggestion
 - type: checkboxes
    id: severity
    attributes:
      label: Severity
      description: Is this blocking customers? How many instances are affected? Is there some context that will help us prioritize?
      options:
        - label: Functionality is severely affected
        - label: Blocking an internal or external customers
        - label: Many customers (or potentially many)
        - label: No known workaround
        - label: Preventing upgrade
```